### DISCUSSION TITLE
Will we run into issues when a field is an array?

### DISCUSSION THREAD

**Discussion by brianorwhatever** [decentralized-identity/waci-didcomm#19] [Q&A]
Speaking with @swcurran he mentioned that aca-py builds the frame object from input descriptors on the prover side. One of the reasons he cited for that approach was because we are going to encounter issues when we have a field thats value is an array. This is because of the index of the message being unknown at the time of request. Do we think this might be an issue?

I am unable to test as I'm still debugging my derivation implementation with a simple example.

**Comment by OR13** [comment]
Array size and order might impact deterministic generation of JSON-LD frame from input descriptors. 

We still don't have code that confirms this is possible to do, but i think there is some in ACA-Py

**Comment by OR13** [comment]
Not sure what to do with this issue, suggest closing as it is stale, and there is no engagement.

**Comment by OR13** [comment]
Move to discussion maybe...
